### PR TITLE
Invalid API description for Lead source API (GWY-14895)

### PR DESCRIPTION
**Pull request by vykhrystiuk**
## Details

LeadSource POST is incorrect and unused it should be renamed to PUT

### CODE REVIEW

File: openapi/paths/customers@{id}@lead-source.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ get:
     - lang: PHP
       source:
         $ref: '../code_samples/PHP/customers@{id}@lead-source/get.php'
-post:
+put:
```
Comment (VladimirBerdnik) [inline openapi/paths/customers@{id}@lead-source.yaml]: Should https://github.com/Rebilly/RebillyAPI/pull/809/files#diff-768ada634c99e1fea3017b424f3bd745R37

and https://github.com/Rebilly/RebillyAPI/pull/809/files#diff-768ada634c99e1fea3017b424f3bd745R68

be updated as well?

Comment (VladimirBerdnik) [reply openapi/paths/customers@{id}@lead-source.yaml]: Ok, I see, changes were just done

Comment (vykhrystiuk) [reply openapi/paths/customers@{id}@lead-source.yaml]: I think so. Updated

**Comment by vykhrystiuk** [comment]
php please review

**Pull request merged**